dev/ci/ci-coq_performance_tests.sh

```diff
@@ -5,4 +5,9 @@ ci_dir="$(dirname "$0")"
 
 git_download coq_performance_tests
 
-( cd "${CI_BUILD_DIR}/coq_performance_tests" && make coq perf-Sanity && make validate && make install )
+# run make -k; make again if make fails so that the failing file comes last, so that it's easier to find the error messages in the CI log
+function make_full() {
+    make "$@" || { RV=$?; make -k "$@"; make "$@"; return $RV; }
```

Comment:
Why not just 
```suggestion
    make -k "$@" || make "$@"
```
?
Or even twice with -k so that you get all the errors

Reply:
In exceptional cases, `make -k; make` will succeed in cases where `make` does not.  (For example, if dependences are improperly computed.). Perhaps this doesn't matter on Coq's CI, as it should be tested on the CI of coq-performance-tests, but it seems better to be safe.  I can add a comment to this effect if that's desired?

Reply:
What about `if ! make -k; then make -k; exit 1; fi`?

Reply:
(second `-k` optional)

Reply:
Doesn't `make -k` always succeed?

Reply:
No, it continue after an error but still fails if there is one